packages/core/src/events/handlers/documentLogs/createJob.ts

```diff
@@ -31,4 +34,14 @@ export const createDocumentLogJob = async ({
       uuid: event.data.documentLogUuid,
     },
   }).then((r) => r.unwrap())
+
+  websockets.emit('documentLogCreated', {
+    workspaceId,
+    data: {
+      workspaceId,
+      documentUuid: event.data.documentUuid,
+      commitUuid: event.data.commitUuid,
+      documentLogId: documentLog.id,
+    },
+  })
```

Comment:
side effect should be in an event handler, we already have the documentLogCreated event in fact. Leave a TODO if you don't wanna do it now.

Reply:
But this is the `DocumentRunEvent`'s event handler.

Reply:
sure but this is a side effect of document log created event, instead of having a single job that handles all side effects it's more maintainable and it scales better to have an event handler for each side effect